### PR TITLE
Use structure to fit methodology

### PR DESCRIPTION
**Pull request by razvand**
Update repository structure to fit the structure for content repositories presented in the [`methodology`](https://github.com/open-education-hub/methodology/)
    
In summary this means:
    
- Remove the `content/` directory. The top-level directory is `chapters/`.
- Topics gets their own directory inside a chapter directory. Add a directory for each chapter topic.
- In each topic directory, there are directories for types of content. Add `reading/` type of content in each topic directory. Add blank `README.md` in each `reading/` type of content.
    
Also update `config.yaml` accordingly.

Update `Dockerfile` and introduce `Makefile`.

Add global `.gitignore` file.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.PHONY: all buildimg build serve run_bash enter_bash stop_bash clean cleanall
+
+all: build
+
+buildimg:
+	docker build -f ./Dockerfile --tag $(IMAGE_NAME) .
+
+build: buildimg
+	@echo "Building content. This will take a while (several minutes) ..."
+	@echo "After the build, run `make serve`"
```
Comment (teodutu) [inline Makefile]: `make serve` is not needed. Running the container already starts serving port 8080. No idea why, though.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	@mkdir -p $(OUTPUT_DIR)
+	docker run --rm -v $$PWD/:/content -v $(OUTPUT_DIR):/output $(IMAGE_NAME)
+
+serve:
+	@echo "Point your browser to http://localhost:8080/$(REPO_NAME)"
+	@cd $(OUTPUT_DIR)/.. && python -m http.server 8080
```
Comment (teodutu) [inline Makefile]:
```suggestion
	@cd $(OUTPUT_DIR)/.. && python3 -m http.server 8080
```

**Review by teodutu — CHANGES_REQUESTED** [review]
This suffers from the same issue with `config.yml` as https://github.com/open-education-hub/methodology/pull/28. Both will be fixed when the builder can support paths as subsections.

File: Makefile (hunk 1 above)
Comment (razvand) [reply Makefile]: Are you sure? Does it work for you?

Comment (teodutu) [reply Makefile]: Yes. I ran your PR locally and that's how I noticed this.

**Review by teodutu — APPROVED** [review]
The Makefile works fine and the empty content is rendered nicely.

**Review by gabrielmocanu — APPROVED** [review]

**Pull request merged**